apps/ledger-live-desktop/src/renderer/modals/Send/fields/DomainErrorHandlers.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { memo } from "react";
+import { InvalidDomain, NoResolution } from "@ledgerhq/domain-service/errors/index";
+import { DomainServiceResponseError } from "@ledgerhq/domain-service/hooks/types";
+import Alert from "~/renderer/components/Alert";
+import { useTranslation } from "react-i18next";
+
+type DomainErrorsProps = {
+  domainError: DomainServiceResponseError;
+  isForwardResolution: boolean;
+};
+export const DomainErrorsView = memo(({ domainError, isForwardResolution }: DomainErrorsProps) => {
+  const { t } = useTranslation();
+  if ((domainError.error as Error) instanceof InvalidDomain) {
```

Comment:
ok, i see what you refer to, here is a pattern i found interesting to workaround:

```ts
function isErrorKindOfA (error: unknown) {
  return (error instanceof FooError || error instanceof BarError)
}
```

like, you can get back to "unknown" and it will be accepted